### PR TITLE
[15.0] Migration of purchase_analytic

### PR DESCRIPTION
**Pull request by alan196**
### Proposed changes

Migration PR for module purchase_analytic.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the necessary documentation

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
Code review LGTM

**Comment by rousseldenis** [comment]
@alan196 Could you rebase ?

**Comment by alan196** [comment]
@AaronHForgeFlow @rousseldenis Rebased 👍

**Comment by alan196** [comment]
I also allowed @OCA-git-bot to write in Jarsa-dev repositories, who accepts the invitation to the bot?

**Review by rousseldenis — APPROVED** [review]

File: purchase_analytic/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+13.0.1.0.0 (2020-01-08)
```
Comment (AaronHForgeFlow) [inline purchase_analytic/readme/HISTORY.rst:6]: Can you add this PR to the history?

**Review by AaronHForgeFlow — APPROVED** [review]
Functional + code, just a minor comment

File: purchase_analytic/readme/HISTORY.rst (hunk 1 above)
Comment (alan196) [reply purchase_analytic/readme/HISTORY.rst:6]: @AaronHForgeFlow the change is OK?

**Comment by AaronHForgeFlow** [comment]
Yes, thanks

/ocabot merge nobump

**Pull request merged**